<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="20801" started-at="2020-05-08T03:53:13Z" finished-at="2020-05-08T03:53:34Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="20801" started-at="2020-05-08T03:53:13Z" finished-at="2020-05-08T03:53:34Z">
      <class name="com.qa.webDotCom.extentreports.BasePage">
        <test-method status="PASS" signature="init()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="init" is-config="true" duration-ms="13" started-at="2020-05-08T03:53:13Z" finished-at="2020-05-08T03:53:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="loaswebPage()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="loaswebPage" is-config="true" duration-ms="6454" started-at="2020-05-08T03:53:13Z" finished-at="2020-05-08T03:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loaswebPage -->
        <test-method status="PASS" signature="testPassed()[pri:1, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="testPassed" duration-ms="19" started-at="2020-05-08T03:53:20Z" finished-at="2020-05-08T03:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPassed -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="tearDown" is-config="true" duration-ms="617" started-at="2020-05-08T03:53:20Z" finished-at="2020-05-08T03:53:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="loaswebPage()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="loaswebPage" is-config="true" duration-ms="5874" started-at="2020-05-08T03:53:21Z" finished-at="2020-05-08T03:53:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loaswebPage -->
        <test-method status="SKIP" signature="testSkipped()[pri:2, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="testSkipped" duration-ms="6" started-at="2020-05-08T03:53:26Z" finished-at="2020-05-08T03:53:26Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skipped by me skipped method]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skipped by me skipped method
at com.qa.webDotCom.extentreports.BasePage.testSkipped(BasePage.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSkipped -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="tearDown" is-config="true" duration-ms="620" started-at="2020-05-08T03:53:26Z" finished-at="2020-05-08T03:53:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="loaswebPage()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="loaswebPage" is-config="true" duration-ms="6095" started-at="2020-05-08T03:53:27Z" finished-at="2020-05-08T03:53:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loaswebPage -->
        <test-method status="FAIL" signature="testFailed()[pri:3, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="testFailed" duration-ms="4" started-at="2020-05-08T03:53:33Z" finished-at="2020-05-08T03:53:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.qa.webDotCom.extentreports.BasePage.testFailed(BasePage.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFailed -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.webDotCom.extentreports.BasePage@1d9b7cce]" name="tearDown" is-config="true" duration-ms="624" started-at="2020-05-08T03:53:34Z" finished-at="2020-05-08T03:53:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.webDotCom.extentreports.BasePage -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
